plugins/nobbz/lua/nobbz/noice.lua

```diff
@@ -1,3 +1,7 @@
+require("notify").setup({
+  background_color = "#000000",
+})
+
```

Comment:
Hi @coderabbitai, this seems to have slipped in, and is not directly related to this PR, can you please warn me in the future?